decidim-proposals/spec/system/admin/admin_manages_participatory_texts_spec.rb

```diff
@@ -46,81 +46,46 @@ def import_document
     click_on "Upload document"
     expect(page).to have_content "The following sections have been converted to proposals. Now you can review and adjust them before publishing."
     expect(page).to have_content "Preview participatory text"
-    validate_proposals
-  end
-
-  def validate_occurrences(sections: nil, subsections: nil, articles: nil)
-    expect(page).to have_content "Section:", count: sections if sections
-    expect(page).to have_content "Subsection:", count: subsections if subsections
-    expect(page).to have_content "Article", count: articles if articles
-  end
-
-  def move_some_sections; end
 
-  def publish_participatory_text
-    click_on("Publish document")
-    expect(page).to have_content "All proposals have been published"
-  end
-
-  def validate_published
-    proposals = Decidim::Proposals::Proposal.where(component: current_component)
-    titles = [
-      "The great title for a new law",
-      "A co-creation process to create creative creations",
-      "1", "2",
-      "Creative consensus for the Creation",
-      "3", "4", "5",
-      "Creation accountability",
-      "6",
-      "What should be accounted",
-      "7", "8",
-      "Following up accounted results",
-      "9", "10", "11", "12", "13",
-      "Summary",
-      "14", "15"
-    ]
-    expect(proposals.count).to eq(titles.size)
-    expect(proposals.published.count).to eq(titles.size)
-    expect(proposals.published.order(:position).pluck(:title).map(&:values).map(&:first)).to eq(titles)
-  end
-
-  def validate_proposals
     proposals = Decidim::Proposals::Proposal.where(component: current_component)
     proposals.each do |proposal|
       expect(proposal.title).to be_a(Hash)
       expect(proposal.body).to be_a(Hash)
     end
   end
 
-  def edit_participatory_text_body(index, new_body)
-    fill_in(
-      "preview_participatory_text_proposals_attributes_#{index}_body",
-      with: new_body
-    )
-  end
-
-  def save_participatory_text_drafts
-    click_on "Save draft"
-    expect(page).to have_content "Participatory text successfully updated."
-    expect(page).to have_content "Preview participatory text"
-  end
-
-  def discard_participatory_text_drafts
-    accept_confirm "Are you sure to discard the whole participatory text draft?" do
-      click_on "Discard all"
-    end
-    expect(page).to have_content "All participatory text drafts have been discarded."
-    expect(page).to have_content "Preview participatory text"
-  end
-
   describe "importing participatory texts from a document" do
     it "creates proposals" do
       visit_participatory_texts
       import_document
```

Comment:
Let's inline this method too. I prefer to not having to jump to the method definition in a spec if it is being called only once.